docs/advanced_examples/XGBClassifier.ipynb

```diff
@@ -552,33 +538,25 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Execution time: 1.36 seconds per sample\n"
+      "FHE execution time: 0.72 seconds per sample\n"
```

Comment:
did we refresh our notebooks ? I haven't change anything here on the ML side but still got a x2 🤔